backend/src/apiserver/server/test/arguments-parameters.yaml

```diff
@@ -23,6 +23,7 @@ spec:
     - name: param1
       value: hello
     - name: param2
+      value: "world"
```

Comment:
This is the tricky problem I mentioned, this workflow template is valid, because KFP allows injecting parameter values at pipeline execution time.

So I think we might need to move the validation logic to right before running a pipeline, but in that case, argo should already be able to validate the workflow and inform us about the error.

This is tricky, your thoughts?

Reply:
Yes, I actually missed your point previously and had to figure it out myself that it was an issue, :( . I think the easiest workaround for it will be to use the `validatopts` better in argo, 
https://github.com/argoproj/argo-workflows/blob/0a1bff19d066b0f1b839d8edeada819c0f08da57/workflow/validate/validate.go#L35.  I will play around with it and see if we can have linting both at upload and submission.

Reply:
Updated it now to handled validation of workflows that are uploaded and run separately, will go through it tomorrow again and update some of the naming and check it again.